# Request a grasp plan visualization
# This service is used by the panda_autograsp_server to
# evoke the moveit_planner_server VizualizePlan service.

---
bool success
